app/styles/app/layouts/insights.scss

```diff
@@ -1,339 +1,341 @@
+  }
+}
+
+.insights-overlay {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  text-align: center;
+  display: none;
+  font-size: 18px;
+
+  &--active {
+    display: block;
+  }
+
+  .overlay-backdrop {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-flow: column nowrap;
+  }
+
+  h2, .insights-overlay__header {
+    color: $oxide-blue;
+    font-size: 40px;
+    font-weight: 300;
+    margin: 0 0 15px 0;
+  }
+
+  p {
+    padding: 0 10%;
+    font-size: 18px;
+  }
+}
+
+.insights-glance {
+  padding: 20px 20px 15px;
+  overflow: hidden;
+  width: 100%;
+
+  // This use of grid is just to make some things align better in certain cases. Nonessential and should look fine without it.
+  display: grid;
+  align-items: start;
+  grid-template-columns: 100%;
+  grid-template-rows: auto auto 1fr;
+
+  &__title {
+    font-size: 12px;
+    font-weight: 600;
+    margin: 0;
+    text-transform: uppercase;
+  }
+
+  &__center {
+    display: flex;
+    align-items: center;
+    flex-flow: row wrap;
+  }
+
+  &__stat {
+    font-size: 27px;
+    font-weight: 300;
+    color: $cement-grey;
+
+    .insights-glance__center & {
+      margin-right: 8px;
+    }
+
+    .loading-indicator {
+      font-size: 9px;
+    }
+  }
+
+  &-delta {
+    border-radius: 3px;
+    padding: 0 5px;
+    font-size: 12px;
+    background-color: $pebble-grey;
+    cursor: default;
+
+    svg {
+      width: 10px;
+      height: 10px;
+      margin: 0;
+    }
+
+    &[data-dir='+'] {
+      color: $turf-green;
+      background-color: $seed-green;
+      svg {
+        transform: rotate(180deg);
+        path {
+          stroke: $turf-green;
+        }
+      }
+    }
+
+    &[data-dir='-'] {
+      color: $brick-red;
+      background-color: $quartz-red;
+      svg path {
+        stroke: $brick-red;
+      }
+    }
+  }
+
+  &__chart-placeholder {
+    margin: 33px 0 12px 0;
+    border: none;
+    border-bottom: 1px solid transparent;
+  }
+
+  &__chart {
+    align-self: end;
+  }
+
+  &__centerline line {
+    stroke: #eaeaea;
+  }
+}
+
+.insights-odyssey {
+  overflow: hidden;
+  position: relative;
+  width: 100%;
+
+  &__title {
+    font-size: 15px;
+    font-weight: 400;
+    margin: 0 0 5px 0;
+  }
+
+  &--loading &__chart {
+    padding-bottom: 35%;
+    background-color: $grid-background-color;
+    background-image: linear-gradient(to bottom, rgba($dry-cement, 1) 0px,rgba($dry-cement, 0) 1px,rgba($dry-cement, 0) 60px);
+    background-size: 100% 60px;
+    background-repeat: repeat;
+  }
+
+  &__loading-indicator {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    .loading-indicator i {
+      width: 20px;
+      height: 20px;
+    }
+  }
+
+  // Odyssey C3 customizations
+  .c3-axis {
+    .domain {
+      stroke: $axis-line-color;
+    }
+    .tick {
+      @include snapshotHide;
```

Comment:
This line here is the actual intended change for this file.

Ugh, I guess some of these files had CRLF line endings. I did some dev through Windows Subsystem for Linux, and I thought I had my editor set to always do LF line endings, but I guess not. Anyway, I've recently altered my setup such that I'm going to always be using my macbook. And I'm thinking I'll open up a PR to convert any other files that have CRLF line endings.